plugins/woocommerce-blocks/tests/e2e/tests/all-reviews/all-reviews.block_theme.spec.ts

```diff
@@ -8,55 +8,78 @@ import { expect, test } from '@woocommerce/e2e-playwright-utils';
  */
 import { allReviews } from '../../test-data/data/data';
 
-const blockData = {
-	name: 'woocommerce/all-reviews',
-	selectors: {
-		frontend: {
-			firstReview:
-				'.wc-block-review-list-item__item:first-child .wc-block-review-list-item__text p',
-		},
-	},
-};
-
-test.describe( `${ blockData.name } Block`, () => {
-	test( 'block can be inserted and it successfully renders a review in the editor and the frontend', async ( {
-		admin,
-		editor,
+const BLOCK_NAME = 'woocommerce/all-reviews';
+
+const latestReview = allReviews[ allReviews.length - 1 ];
+
+test.describe( `${ BLOCK_NAME } Block`, () => {
+	test.beforeEach( async ( { admin, editor } ) => {
+		await admin.createNewPost();
+		await editor.insertBlock( { name: BLOCK_NAME } );
+	} );
+
+	test( 'block can be inserted and it sorts reviews by most recent by default', async ( {
+		frontendUtils,
 		page,
 		editorUtils,
 	} ) => {
-		await admin.createNewPost();
-		await editor.insertBlock( { name: blockData.name } );
-
 		await expect( page.getByText( allReviews[ 0 ].review ) ).toBeVisible();
 
 		await editorUtils.publishAndVisitPost();
 
-		await expect( page.getByText( allReviews[ 0 ].review ) ).toBeVisible();
+		const block = await frontendUtils.getBlockByName( BLOCK_NAME );
+		const reviews = block.locator(
+			'.wc-block-components-review-list-item__text'
```

Comment:
I've also opted not to add any labels or attributes to the blocks themselves, but these selectors are less fragile than the ones that were in this test before.